autoreject/utils.py

```diff
@@ -461,3 +461,15 @@ def _fast_map_meg_channels(info, pick_from, pick_to,
     mne.set_log_level(verbose)
 
     return fmd['data']
+
+
+def _patch_setup_dots(mode, info, coils, ch):
+    """
+    Monkey patcher for MNE's _setup_dots (changed parametrization in v0.24).
+    """
```

Comment:
too late but for the future: https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

They really should fit in one line :-)